av_metrics/benches/bench.rs

```diff
@@ -62,19 +63,21 @@ pub fn msssim_benchmark(c: &mut Criterion) {
 pub fn ciede2000_nosimd_benchmark(c: &mut Criterion) {
     let frame1 = get_video_frame::<u8>("./testfiles/yuv420p8_input.y4m");
     let frame2 = get_video_frame::<u8>("./testfiles/yuv420p8_output.y4m");
+    env::set_var("AV_METRICS_DISABLE_SIMD", "1");
```

Comment:
I really dislike this.

Reply:
I'm not a fan either. I think the workaround would be to make a separate module to expose the private structs when benchmarks are enabled, like rav1e does. I'm not a fan of that approach either, but it is better than this. Can you think of any better options?

Reply:
I'd have a feature to expose to criterion the internals and set them as needed. It is a workaround to the fact benchmarks do not carry a specific feature (and we should probably check the issue about it on cargo).